gateway/testutil.go

```diff
@@ -27,7 +27,6 @@ import (
 	"github.com/TykTechnologies/tyk/rpc"
 
 	jwt "github.com/dgrijalva/jwt-go"
```

Comment:
This is unmaintained and vulnerable, this is In-place replacement https://github.com/golang-jwt/jwt

Reply:
Not just a simple search/replace, it has wider impact. See TT-4955 issue and comment. I tried.